src/amoc_api_node_handler.erl

```diff
@@ -9,61 +9,42 @@
 -export([rest_init/2,
```

Comment:
Why this PR touches the node handler? Do not merge changes but rebase your work on top of simple_api2 HEAD so that only your changes are visible in the PR.